docs/installation.rst

```diff
@@ -73,4 +73,4 @@ and then run the following in the terminal:
    $ git remote add upstream git@github.com:tardis-sn/stardis.git
    $ git fetch upstream
    $ git checkout upstream/main
-   $ pip install -e .[test,docs]
+   $ pip install -e .[test,docs]
```

Comment:
Is there any change here? Do you want to unstage this file since the before and after diffs are identical?